doc/releases/changelog-dev.md

```diff
@@ -4,9 +4,7 @@
 
 <h3>New features since last release</h3>
 
-* The `sample_state` function is added to `devices/qubit` that returns a series of samples based on a given
-  state vector and a number of shots.
-  [(#3720)](https://github.com/PennyLaneAI/pennylane/pull/3720)
+<h4>Pulse programming</h4>
 
 * Added the needed functions and classes to simulate an ensemble of Rydberg atoms:
```

Comment:
Need some code examples here for sure

Reply:
This is more placeholder for now, it needs to be replaced with a more catchy description of what's been done.